src/components/signup/SignupState.js

```diff
@@ -79,9 +80,19 @@ const Signup = ({ navigation, screenProps }: { navigation: any, screenProps: any
       }, 300)
     }
   }
-  const fireSignupEvent = (event?: string) => {
+
+  const fireSignupEvent = async (event?: string) => {
```

Comment:
all changes to this method are not necessaray.
I asked just to verify that each step in signup calls this method

Reply:
changes were made to the method for registration with W3